### ISSUE TITLE
AliasOrder throws false positive

### ISSUE THREAD

**Issue by fritsypeij** [rrrene/credo#575]
### Produces an incorrect error
```
alias Detroit.Learnables.Learnable
alias DetroitWeb.{ContainerCell, WizardNavigationCell}
alias DetroitWeb.Course.Subject.{CompletionCell, HeaderCell}
```
> The alias `DetroitWeb.ContainerCell` is not alphabetically ordered among its group.

### Passes correctly
```
  alias Detroit.Learnables.Learnable

  alias DetroitWeb.{
    ContainerCell,
    WizardNavigationCell
  }

  alias DetroitWeb.Course.Subject.{
    CompletionCell,
    HeaderCell
  }
```

**Comment by rrrene** [comment]
Thx for reporting! :+1:

I added a test reproducing this issue!

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by pdobacz** [comment]
Hi, I might be getting a related error with:

```
  alias OMG.API.State.{Transaction, Transaction.Recovered, Transaction.Signed}
  alias OMG.API.Utxo
  require Utxo
  alias OMG.Watcher.Repo
```

I am however unable to arrange these so that I don't get a complaint. It worked with `0.10.0`, It works with `fdb8782cdac40d49824e973346ba2aa341427240`

**Comment by rrrene** [comment]
@pdobacz Ouch. It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by navinpeiris** [comment]
hi @rrrene, was having the same kind of issue as @pdobacz, can confirm that it's been fixed now
